### PR TITLE
feat: added Navigation

### PR DESCRIPTION
**Pull request by Prathyusha-96**
image
image
image

### CODE REVIEW

**Review by aamir74 — APPROVED** [review]
-can remove extra white space lines in css file
-can format the html code for better readability (use formatters like prettier)
Rest looks good

File: components/Navigation/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    border-radius: 5px;
+    font-family: var(--font-family);
+  }
+
+  .navigation-heading {
+    font-size: 1.5rem;
+    color: var(--text-color);
+  }
+
+
```
Comment (Meghana-sk) [inline components/Navigation/Navigation.css:16]: Extra lines can be removed as it increases lines of code

File: components/Navigation/Navigation.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  .navbar-search {
+    display: flex;
+  }
+
+  .searchbar {
+    width: 100%;
+    border-width: 0;
+    padding: 0.5rem;
+    border-radius: 5px;
```
Comment (Meghana-sk) [inline components/Navigation/Navigation.css:47]: Suggestion: A variable can be used for border radius

**Review by Meghana-sk — CHANGES_REQUESTED** [review]
1. Since all the components have folder structure with small case,  for navigation it is ### `components/Navigation/Navigation.html` but other folders `components/alert/alert.html` . It will be good if the folder naming is uniform.
2. Cursor pointer can be added for brand name as well because clicking on brand name usually takes back to home page.

**Pull request merged**